### ISSUE TITLE
Phoenix tracing aborts chef-run

### ISSUE THREAD

**Issue by pu239ppy** [bloomberg/chef-bach#493]
chef-client exists with a long stack trace that boils down to the fact that SYSTEM.TRACING_STATS already exists
````
^[[1;31mAborting command set because "force" is false and command failed: "CREATE TABLE SYSTEM.TRACING_STATS (trace_id BIGINT NOT NULL, parent_id BIGINT NOT NULL, span_id BIGINT NOT NULL, description VARCHAR, start_time BIGINT, end_time BIGINT, hostname VARCHAR, tags.count SMALLINT, annotations.count SMALLINT, CONSTRAINT pk PRIMARY KEY (trace_id, parent_id, span_id));"^[[m
Closing: org.apache.phoenix.jdbc.PhoenixConnection
---- End output of "bash"  "/tmp/chef-script20160412-15193-1srstf8" ----
Ran "bash"  "/tmp/chef-script20160412-15193-1srstf8" returned 2
[2016-04-12T06:15:59-04:00] ERROR: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
````

**Comment by bijugs** [comment]
The changes to enable Phoenix tracing was idempotent on HDP 2.2 where the original changes were made. Looks like the behavior of ``sqlline.py`` from Phoenix must have changed in the later version included in HDP 2.3.x. Will look into this.

**Comment by bijugs** [comment]
Looking ``sqlline.py`` code in HDP repo [2.2](https://github.com/hortonworks/phoenix/blob/HDP-2.2.0.0-tag/bin/sqlline.py) and [2.3.4](https://github.com/hortonworks/phoenix/blob/HDP-2.3.4.1/bin/sqlline.py), there has been a change in the way return code is [propagated up to the shell](https://github.com/hortonworks/phoenix/blob/HDP-2.3.4.1/bin/sqlline.py#L69-L71). This change is also made available on Apache Phoenix 4.6 as part of [PHOENIX-2323](https://issues.apache.org/jira/browse/PHOENIX-2323). This change lets us to conditionally run "CREATE TABLE ..."

**Comment by bijugs** [comment]
Resolved in PR #496 .
